apps/dcos_net/src/dcos_net_killer.erl

```diff
@@ -70,6 +71,41 @@ reductions() ->
         end
     end, #{}, erlang:processes()).
 
+-spec(maybe_kill_dist_proxy() -> true | no_return()).
+maybe_kill_dist_proxy() ->
+    Pid = whereis(ssl_tls_dist_proxy),
+    maybe_kill_dist_proxy(Pid).
+
+-spec(maybe_kill_dist_proxy(undefined | pid()) -> true | no_return()).
+maybe_kill_dist_proxy(undefined) ->
+    true;
+maybe_kill_dist_proxy(Pid) ->
+    case erlang:process_info(Pid, messages) of
+        {messages, Messages} ->
+            maybe_kill_dist_proxy(Pid, Messages);
+        undefined ->
+            true
+    end.
+
+-spec(maybe_kill_dist_proxy(pid(), [term()]) -> true | no_return()).
+maybe_kill_dist_proxy(Pid, Messages) ->
+    MaxMsgs = application:get_env(dcos_net, ssl_tls_dist_proxy_max_msgs, 64),
```

Comment:
What is the rationale behind 64 messages?

Reply:
On soak cluster `ssl_tls_dist_proxy` receives a connection request every 10 seconds or so when it hits this issue. So in 10-15 minutes the killer will stop the whole vm and systemd restart it.